### ISSUE TITLE
Separate text and rodata for armv8

### ISSUE THREAD

**Issue by Sankyou** [openssl/openssl#23911]
To improve performance, OpenSSL uses assembly to implement some encryption algorithms when add cflags like 
`-DECP_NISTZ256_ASM`, `-DKECCAK1600_ASM`, `-DOPENSSL_BN_ASM_MONT`, `-DSHA256_ASM`...
And some data is stored in .text section, which is conflict for eXecute-Only-Memory (XOM) feature, because XOM will set permission of .text to executable only, read is not allowed, and data need be read.
So can we move data to .rodata section in this kind of assembly code to support XOM feature?

**Comment by Sankyou** [comment]
@t8m Do you mean the feature is already supported in master branch?

**Comment by t8m** [comment]
No, it means that the target branch for the feature would be the master branch.

**Comment by t8m** [comment]
Please note we are looking for community help implementing this issue. Please comment if you are interested in implementing this feature or if you have reasons why this feature is important enough so we should consider implementing it with higher priority than other requested features. The issue will be closed after 2 months otherwise.

**Comment by t8m** [comment]
This issue was closed because it was marked with the help wanted label and there was no interest in implementing this issue or any other activity leading to implementing or re-prioritizing it for more than 2 months.
